packages/arb-avm-cpp/data_storage/src/arbcore.cpp

```diff
@@ -1199,9 +1239,8 @@ rocksdb::Status ArbCore::reorgCheckpoints(
         last_machine = std::make_unique<Machine>(*core_machine);
     }
 
-    // Checkpoint was saved at sideload, attempt to continue running
+    // Remove any extra messages left in machine from reorg
     core_machine->machine_state.context.clearInboxMessages();
-    core_machine->continueRunningMachine(true);
```

Comment:
Not sure why this was removed?

Reply:
advanceCoreToTarget already called continueRunningMachine() in a loop until there were no sideloads, so this call is not needed

Reply:
I might be misremembering that code, but I think it sets a gas limit to ensure we don't run into reorg'd out messages. I think we have the continueRunningMachine after clearing inbox messages to finish up any extra processing.

Reply:
interesting, I'll put it back in with comment to that effect